helm/dashboards/dashboards/mixin/docker-io-images.json

```diff
@@ -0,0 +1,414 @@
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": [
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Image"
+            },
+            "properties": [
+              {
+                "id": "custom.width",
+                "value": 314
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 13,
+        "w": 24,
+        "x": 0,
+        "y": 25
+      },
+      "id": 8,
+      "interval": "1m",
+      "options": {
+        "cellHeight": "sm",
+        "footer": {
+          "countRows": false,
+          "fields": "",
+          "reducer": [
+            "sum"
+          ],
+          "show": false
+        },
+        "showHeader": true,
+        "sortBy": [
+          {
+            "desc": false,
+            "displayName": "Image"
+          }
+        ]
+      },
+      "pluginVersion": "10.1.5",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "sum(kube_pod_container_info{image=~\"docker.io.*\",image!~\".*/giantswarm/.*\",cluster_type=\"workload_cluster\"} or kube_pod_init_container_info{image=~\"docker.io.*\",image!~\".*/giantswarm/.*\",cluster_type=\"workload_cluster\"}) by (image)",
+          "format": "table",
+          "instant": true,
+          "legendFormat": "Total",
+          "range": false,
+          "refId": "A"
+        }
+      ],
+      "title": "Image list",
+      "transformations": [
+        {
+          "id": "organize",
+          "options": {
+            "excludeByName": {
+              "Time": true
+            },
+            "indexByName": {
+              "Time": 6,
+              "Value": 5,
+              "cluster_id": 2,
+              "container": 4,
+              "image": 0,
+              "namespace": 3,
+              "organization": 1
+            },
+            "renameByName": {
+              "Time": "",
+              "Value": "Container count",
+              "cluster_id": "Cluster",
+              "container": "Container name",
+              "image": "Image",
+              "namespace": "Namespace",
+              "organization": "Organization"
+            }
+          }
+        }
+      ],
+      "type": "table"
+    }
+  ],
+  "refresh": "",
+  "schemaVersion": 38,
+  "style": "dark",
+  "tags": [
+    "owner:team-honeybadger"
+  ],
+  "templating": {
+    "list": []
+  },
+  "time": {
+    "from": "now-3h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "utc",
+  "title": "Container images from docker.io",
+  "uid": "be4cd321-9c4c-4447-b060-fc875c089f5e",
```

Comment:
Question to @giantswarm/team-atlas : do we prefer a friendlier uid here, e. g. `docker-io-images`? I see various ways in the repo.

Reply:
We do not. But maybe make it container-images so we can add other sources later?

Reply:
Friendly `uid`'s are welcome :)

Reply:
You may prefer a friendlier uid especially if you want to add a link to this dashboard from alerts.

Reply:
ACK, done